content/applications/finance/accounting/customer_invoices/deferred_revenues.rst

```diff
@@ -43,9 +43,20 @@ Generate Entries
   :ref:`generate them manually <customer_invoices/deferred/generate_manually>` by selecting the
   :guilabel:`Manually & Grouped` option instead.
 Amount Computation
-  Suppose an invoice of $1200 must be deferred over 12 months. The :guilabel:`Equal per month`
-  computation accounts for $100 each month, while the :guilabel:`Based on days` computation
-  accounts for different amounts depending on the number of days in each month.
+  Suppose an invoice of $1200 must be deferred over 12 months.
+
+  - the :guilabel:`Months` option accounts for $100 each month prorated to the number of days in
+    that month (e.g. $50 for the first month if the :guilabel:`Start Date` is set to the 15th of
+    the month);
```

Comment:
```suggestion
  - The :guilabel:`Months` option accounts for $100 each month prorated to the number of days in
    that month (e.g. $50 for the first month if the :guilabel:`Start Date` is set to the 15th of
    the month).
```
Since each bullet item is a full sentence, let's capitalize the first word and put a period at the end